### PR TITLE
Update cabal-plan.cabal

### PR DESCRIPTION
**Pull request by mimi1vx**
cabal-plan builds fine with new versions of this libraries

### CODE REVIEW

File: cabal-plan.cabal [hunk 1]
```diff
@@ -77,7 +77,7 @@ library
                        RecordWildCards
   exposed-modules:     Cabal.Plan
 
-  build-depends:       base              (>= 4.6 && <4.10) || ^>= 4.10 || ^>=4.11 || ^>=4.12 || ^>=4.13 || ^>=4.14
+  build-depends:       base              (>= 4.6 && <4.10) || ^>= 4.10 || ^>=4.11 || ^>=4.12 || ^>=4.13 || ^>=4.14 || ^>=4.15
```
Comment (fgaz) [inline cabal-plan.cabal:80]: this base change probably needs a GHC==9.0.1 tested-with entry too

**Comment by fgaz** [comment]
pinging @phadej

**Comment by phadej** [comment]
GHC-9.0.1 is badly broken, and I'm waiting for GHC-9.0.2 (and maybe also GHC-9.2 first release candidate) to do any base bumping stuff.

**Comment by andreasabel** [comment]
This PR seems now obsolete:
https://github.com/haskell-hvr/cabal-plan/blob/033130730039aa6bc4efacc9be658843454c69a2/cabal-plan.cabal#L77
The latest release works with base-4.15.
Cabal is also bumped to 3.6.2.

**Pull request closed without merging**